src/app/modules/main/controller.nim

```diff
@@ -126,6 +126,14 @@ method init*(self: Controller) =
     var args = ResolvedContactArgs(e)
     self.delegate.resolvedENS(args.pubkey, args.address, args.uuid)
 
+  self.events.on(SIGNAL_CONTACT_UPDATED) do(e: Args):
```

Comment:
@jrainville not sure do we need to maintain users at all from this place (main module)? Actually we don't need to maintain it from communities module either. Maybe I didn't realize the point here, but how I see a use case of communities module is:
- have communities module as a convenient way to work with "Communities" popup
- we don't need to populate it's list on the app start (since one may start the app and doesn't open "Communities" popup at all)
- we should just populate model of communities module and prepare data there only once user clicks on "Chat section -> Plus button -> Communities option", so prepare data on demand
- because of the previous point that further means we don't need to maintain users details at all since model will be created with the most recent details whenever we need to display it